firmware/robot2015/CMakeLists.txt

```diff
@@ -1 +1,7 @@
-add_subdirectory(cpu)
+# we'll go ahead and set the AVR toolchain cmake file too for consisteny in defining all toolchain setup files
+set(ATTINY_TOOLCHAIN_FILE ${CMAKE_CURRENT_LIST_DIR}/attiny_toolchain.cmake)
+set(ATTINY_TOOLCHAIN_FILE ${ATTINY_TOOLCHAIN_FILE} PARENT_SCOPE)
```

Comment:
I think we can delete this line - we shouldn't need to reference the attiny toolchain anywhere above this directory.